coverage.py

```diff
@@ -532,11 +535,11 @@ def _input_identifier_types(cls):
             return value
 
     @classmethod
-    def register(cls, identifier, collection=None):
-        """Registers an identifier for future coverage
+    def register(cls, identifier, collection=None, force=False):
+        """Registers an identifier for future coverage.
 
         This method is primarily for use with CoverageProviders that use the
-        `preregistered_only` flag to process items. It's currently only in use
+        `registered_only` flag to process items. It's currently only in use
         on the Metadata Wrangler.
```

Comment:
`force` doesn't do what I expected an argument called `force` to do. Add a docstring for it and maybe either:

1. call it `force_to_registered`
2. call it `force_status` and make it a string so the caller can choose which status to force

Reply:
Never mind, I confused this method with `add_for`. Boolean `force` makes sense here. But it does need a docstring.